pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[tool.poetry]
+name = "numpy-financial"
+version = "1.1.0.dev0"
+description = "Simple financial functions"
+license = "BSD-3-Clause"
+authors = ["Travis E. Oliphant et al."]
+maintainers = ["Numpy Financial Developers <numpy-discussion@python.org>"]
+readme = "README.md"
+homepage = "https://numpy.org/numpy-financial/latest/"
+repository = "https://github.com/numpy/numpy-financial"
+documentation = "https://numpy.org/numpy-financial/latest/#functions"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Financial and Insurance Industry",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
```

Comment:
Minor: double entry for 3.9 here

Reply:
Python 3.12 betas are already out, so you may as well add that line already; nothing will break anymore for pure Python packages like `numpy-financial`.

Reply:
Thanks for the tip. I'll add it